frontend/javascripts/oxalis/model/reducers/reducer_helpers.ts

```diff
@@ -95,6 +95,7 @@ export function convertServerAnnotationToFrontendAnnotation(annotation: APIAnnot
     tracingStore,
     owner,
     contributors,
+    organization,
```

Comment:
apparently the `organization` prop was already sent by the backend but the frontend discarded this.
-> I fixed this here.

Side note: But there might also be other fields the frontend drops in this function which might be useful to still have :thinking:.

Reply:
> But there might also be other fields the frontend drops in this function which might be useful to still have 🤔.

if the frontend doesn't use these properties, there's no need to add them, I think.